modules/mono/editor/GodotTools/GodotTools/Inspector/InspectorPlugin.cs

```diff
@@ -28,6 +28,15 @@ public override void _ParseBegin(GodotObject godotObject)
                     continue;
 
                 string scriptPath = script.ResourcePath;
+
+                if (string.IsNullOrEmpty(scriptPath))
+                {
+                    // Generic types used empty paths in older versions of Godot
```

Comment:
There is another case that this type belongs to an external assembly and does not have a `ScriptPath`.
Related: #88536

Reply:
This should be fixed by the same patch 🙏

Reply:
The linked issue has been modified. Looking at the editing history, it looks like it was about the InspectorPlugin throwing an exception on scripts with empty paths which would indeed be fixed by the changes in this PR.